daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -340,6 +343,12 @@ commandParser = subparser $ fold
                 , clientMetadataPlugin = Nothing
                 }
 
+    httpJsonFlag :: ShowJsonApi -> Parser LedgerApi
+    httpJsonFlag (ShowJsonApi showJsonApi) = flag GRPC HttpJson $
```

Comment:
What do we gain by only hiding it instead of doing something like `if not showJsonApi then pure GRPC else flag …` so you cannot even use it?

Reply:
agreed, removed the flag for unsupported commands.